generators/python-v2/ast/src/PythonFile.ts

```diff
@@ -99,6 +155,35 @@ export class PythonFile extends AstNode {
             }
         }
 
+        return uniqueReferences;
+    }
+
+    private writeComments(writer: Writer): void {
+        this.comments.forEach((comment) => {
+            comment.write(writer);
+        });
+
+        if (this.comments.length > 0) {
+            writer.newLine();
+        }
+    }
+
+    private getImportName({ writer, reference }: { writer: Writer; reference: Reference }): string {
+        const nameOverride = writer.getRefNameOverride(reference);
```

Comment:
Here is one of two places where we use the state that was added to the `Writer`.